src/Symfony/Component/Console/Input/InputDefinition.php

```diff
@@ -297,6 +307,14 @@ public function hasShortcut(string $name)
         return isset($this->shortcuts[$name]);
     }
 
+    /**
+     * Returns true if an InputOption object exists by negated name.
```

Comment:
```suggestion
     * Returns true if an InputOption object exists and is negatable.
```
?

Reply:
hmm.. if you send a suggestion, it means that the original comment is missleading. But your suggestion is wrong. Please help me finding a better description.

Here is the situation: we have options that are negatable. The negated name of `--foo` is `--no-foo`.

This function, returns true when called with argument `--no-foo`. This comment is very similar to `hasShortcut` method `Returns true if an InputOption object exists by shortcut.`

Reply:
Sorry you are right, I Mixed things up.